Update java-setup.md (#3741)
forcedotcom/salesforcedx-vscode: docs/_articles/en/getting-started/java-setup.md

lang: en
---

> The Apex Language Server, shipped as part of the Salesforce Apex Extension for VS Code depends upon the Java Platform, Standard Edition Development Kit (JDK). It requires an installation of either JDK version 11 (Recommended) or JDK version 8. By default, the extension attempts to locate your local Java installation by looking for a `JAVA_HOME` or `JDK_HOME` environment variable on your computer. If the extension can't find your Java installation, or if you want it to use a different installation, change the `salesforcedx-vscode-apex.java.home` setting.

Reviewer: ```suggestion
The Apex Language Server, shipped as part of the Salesforce Apex Extension for VS Code, depends on the Java Platform, Standard Edition Development Kit (JDK). It requires an installation of either JDK version 11 (Recommended) or JDK version 8. By default, the extension attempts to locate your local Java installation by looking for a `JAVA_HOME` or `JDK_HOME` environment variable on your computer. If the extension can't find your Java installation, or if you want it to use a different installation, change the `salesforcedx-vscode-apex.java.home` setting.
```

1. Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS).
2. Search for `apex`.

> 3. Change the `salesforcedx-vscode-apex.java.home` setting to the full pathname of your Java Runtime. Do **not** set it to the Java executable itself. Note that this path cannot be set to a location inside your project folder.

Reviewer: ```suggestion
3. Change the `salesforcedx-vscode-apex.java.home` setting to the full pathname of your Java Runtime. Do **not** set it to the Java executable itself. Note that you can't set this path to a location inside your project folder.
```

> 3. Change the `salesforcedx-vscode-apex.java.home` setting to the full pathname of your Java Runtime. Do **not** set it to the Java executable itself. Note that this path cannot be set to a location inside your project folder.

Reviewer: ```suggestion
3. Change the `salesforcedx-vscode-apex.java.home` setting to the full pathname of your Java Runtime. Do **not** set it to the Java executable itself. Note that this pathname can't point to a location inside your project folder.
```

1. Click the **Latest Release** button to download.

> 1. Once the file is downloaded open it and complete the installation steps.

Reviewer: ```suggestion
1. After the file is downloaded, open it and complete the installation steps.
```

> Inside Visual Studio Code, you'll need to set the `salesforcedx-vscode-apex.java.home` to one of the following values.

Reviewer: ```suggestion
Inside Visual Studio Code, set the `salesforcedx-vscode-apex.java.home` to one of the following values.
```

1. For Windows Download the "11", "JDK", "Client" version.
1. Once the file is downloaded open it and complete the installation steps.

> Inside Visual Studio Code, you'll need to set the `salesforcedx-vscode-apex.java.home` to one of the following values.

Reviewer: ```suggestion
Inside Visual Studio Code, set the `salesforcedx-vscode-apex.java.home` to one of the following values.
```

}
```

## Oracle Java 11 (Officially Tested)

1. Navigate to the [download page](https://www.oracle.com/technetwork/java/javase/downloads/jdk11-downloads-5066655.html)
1. Click the **Download** button for **Java SE 11.x.y (LTS)**
1. Accept the license agreement
1. Click the download link that applies to your OS
1. For macOS download the dmg version.
1. For Windows Download exe version.

> 1. If prompted to login, you'll need to login with your Oracle account.

Reviewer: ```suggestion
1. If prompted to login, log in with your Oracle account.
```

1. Once the file is downloaded open it and complete the installation steps.

> Inside Visual Studio Code, you'll need to set the `salesforcedx-vscode-apex.java.home` to one of the following values.

Reviewer: ```suggestion
Inside Visual Studio Code, set `salesforcedx-vscode-apex.java.home` to one of the following values.
```
